runbot_travis2docker/models/runbot_branch.py

```diff
@@ -88,10 +89,15 @@ def cron_weblate(self):
                          ('uses_weblate', '=', True)])
                     if has_build:
                         continue
-                    remote = 'wl-%s' % project['slug']
+                    component = [item for item in project['components']
```

Comment:
That change is required because the weblate create only one repository for all components